neurogym/envs/contrib/leverpress.py

```diff
@@ -69,7 +69,7 @@ def _step(self, action):
             self.steps_beyond_done = 0
         else:
             if self.steps_beyond_done == 0:
-                logger.warn(
+                logger.warn(  # FIXME: consider using loguru instead
```

Comment:
Why not change this to the loguru logger already in this PR? Same for the comments below in this file